.ci_support/win_cxx_compilervs2015vc14.yaml

```diff
@@ -1,10 +1,6 @@
 cxx_compiler:
 - vs2015
-openblas:
-- 0.2.20
```

Comment:
This should stay, no? My guess is that re-rendering will re-add those.

Reply:
`openblas` is not used in windows

Reply:
Ah OK. But we do have a working version of openblas on Windows though, right?

Reply:
We have a fallback when no blas version is available. openblas used to not be available on windows.

I will open a separate PR for this.